Formula/convox.rb

```diff
@@ -0,0 +1,15 @@
+class Convox < Formula
+  desc "The convox AWS PaaS CLI tool"
+  homepage "https://convox.com/"
+  url "https://bin.equinox.io/a/9RUvivxx6Lm/convox-20160606232025-darwin-amd64.tar.gz"
```

Comment:
We'll want to build this from source. Homebrew (the formula side) doesn't accept upstream binaries unless they're something like a `.jar`. The [Cask](https://github.com/caskroom/homebrew-cask) takes binaries, if that's what you were thinking of.